client/pages/topsecret.jsx

```diff
@@ -13,31 +63,125 @@ const HomePage = React.createClass({
       console.log(this.context);
     }
 
+
   },
 
   render() {
+    let initialBankSavings = this.state.initialBankSavings;
```

Comment:
If these variables aren't being edited down the road, they should be const instead of let